lxd_image_server/tools/mirror.py

```diff
@@ -47,22 +49,27 @@ def servername(self):
 class MirrorManager():
     img_dir = '/var/www/simplestreams/images'
     mirrors = {}
+    _lock = Lock()
 
     @classmethod
     def update(cls):
         logger.info('Updating all mirrors')
+        mirros = {}
```

Comment:
Typo.

The problem I found with all this it was that it is a bit difficult to unit test.